docs/site/content/en/docs/Concepts/core-concepts/index.md

```diff
@@ -137,3 +138,6 @@ Actions only for a particular [`Test`](#test)
 
 ## Action allow list
 `Horreum` will only allow generic HTTP requests to domains that have been pre-configured in Horreum by an Administrator.
+
+## API Filter Query
+`Horreum` API provides query parameters that are `JSONPath` paths that filter operation results.
```

Comment:
Can we link to the `APIFilterQueryReference.md` doc so users can navigate to more details?

Reply:
sure